config/sync/field.field.media.image.field_media_image.yml

```diff
@@ -22,15 +22,15 @@ settings:
   handler_settings: {  }
   file_directory: '[date:custom:Y]-[date:custom:m]'
   file_extensions: 'png gif jpg jpeg'
-  max_filesize: ''
+  max_filesize: 256mb
```

Comment:
Consider why it is necessary to set this value.

- 256MB seems quite high for a maximum file size
- I would not spot any reasoning in the commit comment.